firstboot/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+install(FILES firstboot.service
+  DESTINATION /etc/systemd/system/
```

```suggestion
  DESTINATION ${CMAKE_INSTALL_ SYSCONFDIR}/systemd/system/
```
Do we want /etc or /lib/systemd?

The main problem will end up in the Debian package. That should use dh_installsystemd and needs the files in the Debian directory. Has anyone tested whether it makes any sense to integrate the unit files into cMake?

---

Other .service files of this project are in "/lib/systemd/system". We should check Debian branch, how the systemd services are handled. If there is no dh magic, we have to write our own postins scripts :(

---

https://unix.stackexchange.com/a/208352

---

/etc/systemd is for the user and where the symlinks are created to enable/disable units. Never install a systemd file into /etc/systemd.